### ISSUE TITLE
Strange crash relating to RichTextEntry/RichTextLabel

### ISSUE THREAD

**Issue by gradientvera** [space-wizards/RobustToolbox#2339]
paging @Efruit because of hot potato

The steps to reproduce this require the latest SS14 content master and engine master as well. 
1. Spawn any item that uses the Item Status Panel, such as stee or, any other stack, multitools such as jaws of life, etc. 
2. Click on the entity.
3. Crash!

This happens consistenly.

```
[FATL] unhandled: System.NullReferenceException: Object reference not set to an instance of an object.
   at Robust.Client.UserInterface.RichTextEntry.Draw(DrawingHandleScreen handle, IFontLibrary font, UIBox2 drawBox, Single verticalOffset, Single uiScale, Color defColor) in /home/vera/Projects/space-station-14/RobustToolbox/Robust.Client/UserInterface/RichTextEntry.cs:line 69
   at Robust.Client.UserInterface.Controls.RichTextLabel.Draw(DrawingHandleScreen handle) in /home/vera/Projects/space-station-14/RobustToolbox/Robust.Client/UserInterface/Controls/RichTextLabel.cs:line 48
   at Robust.Client.UserInterface.Control.DrawInternal(IRenderHandle renderHandle) in /home/vera/Projects/space-station-14/RobustToolbox/Robust.Client/UserInterface/Control.cs:line 471
   at Robust.Client.UserInterface.UserInterfaceManager._render(IRenderHandle renderHandle, Control control, Vector2i position, Color modulate, Nullable`1 scissorBox) in /home/vera/Projects/space-station-14/RobustToolbox/Robust.Client/UserInterface/UserInterfaceManager.cs:line 778
   at Robust.Client.UserInterface.UserInterfaceManager._render(IRenderHandle renderHandle, Control control, Vector2i position, Color modulate, Nullable`1 scissorBox) in /home/vera/Projects/space-station-14/RobustToolbox/Robust.Client/UserInterface/UserInterfaceManager.cs:line 784
```

**Comment by Efruit** [comment]
Fixed in #2340 (Particularly, eac789cae60d2b18a95df7ad45deeea36e7509f5)

**Comment by Efruit** [comment]
The item status panel text is still formatted oddly, but I think it's using default styles instead for some reason.

**Comment by Efruit** [comment]
Should have been fixed with the merge of #2340.

**Comment by Efruit** [comment]
OBE #2363
